docs/best-practice/limit-and-offset-vs-cursor-api-and-hasmore.md

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
Add an index.md file to both folders. This will serve as an introduction. The user will see it when they click the folder in the side menu, so use it to provide context and tell the reader what to expect in this section. It doesn't have to be long.